hooks/boot/06-postgresql-upgrade-extensions.rb

```diff
@@ -12,6 +14,11 @@ def os_needs_postgresql_upgrade?
   def postgresql_upgrade(new_version)
     logger.notice("Performing upgrade of PostgreSQL to #{new_version}")
 
+    start_services(['postgresql'])
+    postgres_list = `runuser -l postgres -c 'psql --list --tuples-only --csv'`
```

Comment:
oh nice!

Reply:
`--csv` is "new" in 12: https://www.postgresql.org/docs/release/12.0/